core/src/main/scala/dagr/core/cmdline/DagrCoreMain.scala

```diff
@@ -228,7 +228,8 @@ class DagrCoreMain extends LazyLogging {
         1
       case Sopt.CommandSuccess(cmd) =>
         unreachable("CommandSuccess should never be returned by parseCommandAndSubCommand.")
-      case Sopt.SubcommandSuccess(dagr, pipeline) =>
+      case Sopt.SubcommandSuccess(dagr, originalPipeline) =>
+        val pipeline = modifyPipeline(dagr, originalPipeline)
```

Comment:
I think we would want to run this after configure but before execute. In the case where we want to get log and script dirs.. they will not have been set up prior to configure.

Reply:
👍 feel free to push to this branch directly